com.mastropaolo.www.tabela.json

```diff
@@ -0,0 +1,56 @@
+{
+    "id" : "com.mastropaolo.www.tabela",
```

Comment:
why is `www` in the app-id?

Reply:
Because for stupid ISP reasons, all the requests that come with the TLD only and not a third level get a cert error (the control panel of the ISP only allows me to put an `A` record for the second-level, and I'm currently hosting the cert etc. on a CNAME). I can do a "better" third level, if desired (e.g. `com.mastropaolo.apps.tabela`). Or I can remove the third level, but then the linter fails.